data/json/monsters/nether.json

```diff
@@ -1154,5 +1154,68 @@
     ],
     "death_function": { "corpse_type": "NO_CORPSE", "message": "Your memory can't decide whether the %s was even there in the first place." },
     "flags": [ "FLIES", "NO_BREATHE", "SEES", "HEARS", "RANGED_ATTACKER" ]
+  },
+  {
+    "id": "mon_archunk_weak",
+    "type": "MONSTER",
+    "name": { "str": "chunk of unknown material", "str_pl": "chunks of unknown material" },
+    "description": "This formless, pitch-black mass wriggles on the ground as it reshapes itself before your eyes.  It slowly drags itself towards you with many tendrils of varied sizes.  As it gets closer to you, its wriggling intensifies as if it reacted to you.",
+    "default_faction": "nether",
+    "species": [ "HORROR" ],
+    "//": "No NETHER or it will run away at low HP",
+    "volume": "62500 ml",
+    "weight": "11110 g",
+    "hp": 750,
+    "symbol": "H",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "anger_triggers": [ "HURT" ],
+    "speed": 75,
+    "armor_bash": 20,
+    "armor_cut": 20,
+    "armor_stab": 20,
+    "armor_bullet": 20,
+    "armor_fire": 20,
+    "armor_elec": 20,
+    "bleed_rate": 0,
+    "special_attacks": [
+      [ "TENTACLE", 2 ],
+      {
+        "type": "spell",
+        "spell_data": { "id": "spawning_zombie_die", "hit_self": true },
```

Comment:
So if I'm understanding this correctly this spell is only doing damage to the monster?   If so I think it would be better if the monster had less hp and didn't hurt itself.  Is there something else being achieved with this spell? Or would you like something else to be achieved?

Reply:
It currently put emphasis on the whole "reacts to live humans" thing but it can be changed for a noise source if needed
Setting their hp at 10% will probably be enough

Reply:
The changes were made